### PR TITLE
Fill6016

### PR DESCRIPTION
**Pull request by gkrintir**
@capalmer85  here they are.

The rate files for PCC & VTX are from ZeroBias1 PD. I am making them for the whole set of PD's.

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 if [ "$1" = "Timber" ]
 then
+    export _JAVA_OPTIONS="-Xss2m"
```
Comment (capalmer85) [inline env.sh:3]: whats this for?

Comment (gkrintir) [reply env.sh:3]: the temporary fix for pytimber to work

fwd. the mail from Riccardo and Danilo ;D

**Comment by gkrintir** [comment]
Hi @capalmer85 the complete rate files now added

**Comment by capalmer85** [comment]
@gkrintir can you commit vertex only fit results with DG as well please?  Then I'll merge.

**Comment by gkrintir** [comment]
@capalmer85 ah we commit also the results ? i intentionally did not e.g. i left to the user to decide from where to make the beam currents (hd5, timber fbct or bqm), which corr to apply etc

so in the current version of PR, its only the minimal info i.e., scan file, rate files & the configurations to use them to produce the fit results

if have also to commit the results, i can do it during weekend (difficult but possible) or then monday ;D

**Comment by capalmer85** [comment]
Ok. Never mind that then. Can you commit the configuration files for vertex
only fits though?

**Comment by gkrintir** [comment]
already there https://github.com/CMS-LUMI-POG/VdMFramework/pull/37/files#diff-9e04a32125b2ccc1c45a8014aa49f342 ;D

**Comment by capalmer85** [comment]
@karacheb Sorry it looks like I have merge it...

**Pull request merged**